generators/server/templates/_pom.xml

```diff
@@ -941,9 +963,6 @@
                     Enable the line below to have remote debugging of your application on port 5005
                     <jvmArguments>-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005</jvmArguments>
                     -->
-                    <arguments>
-                        <argument>--spring.profiles.active=dev</argument>
```

Comment:
I checked, every scenario except passing switches `./mvnw  "-Drun.profiles=no-liquibase,no-swagger"` like this works. So im gonna put setting of this argument back in while merging so that the above also works as per what we have in docs